Export benchmark results (#8)
atmoos/GlassView: README.md

# GlassView

A solution to view and keep track of benchmarking results of dotnet projects.

> This is still a project very much in it's infancy. But when it matures, it is supposed to export [BenchmarkDotNet](https://github.com/dotnet/BenchmarkDotNet) results such that they can be tracked over time. This should - if all goes according to plans -- allow teams to keep track of the performance characteristics of a project. Just as many teams already do with correctness and quality aspects by keeping trach of test results (coverage) and code quality metrics.

Reviewer: ```suggestion
This is still a project very much in it's infancy. But when it matures, it is supposed to export [BenchmarkDotNet](https://github.com/dotnet/BenchmarkDotNet) results such that they can be tracked over time. This should allow teams to keep track of the performance characteristics of a project, similarly like keeping track of the test results (coverage) and code quality metrics.
```

Author: Thanks for the simplification and clearer wording. I adopted it with subtle adjustments.

## Vision

The end product should be some interactive web interface that can run "anywhere" which enables analysis of your benchmark results.

- [ ] Export benchmark results as json to some local directory.
  - [x] Via a library (ToDo #12: nuget pending)
  - [ ] Completely decoupled without any dll dependencies
- [ ] Ingestion into some service(?) that creates (static) views.
- [ ] Interactive web interface.
- [ ] IDE integration (Visual Studio Code)?

> - [ ] Statistics suite with which,

Reviewer: ```suggestion
- [ ] Statistics suite with where,
```

Author: "with which" is IMHO the correct form.
